### PR TITLE
Create new testing page for each team (required token)

### PR DESCRIPTION
**Pull request by phuchieuct93abc**

### CODE REVIEW

File: pacman-server/src/pacman.js [hunk 1]
```diff
@@ -875,7 +882,8 @@ var PACMAN = (function () {
     return {
         "init": init,
         "move": keyDown,
-        "startNewGame": startNewGame
+        "startNewGame": startNewGame,
+        setNumberOfPlayer:setNumberOfPlayer
```
Comment (nthungvn) [inline pacman-server/src/pacman.js]: Why doesn't `setNumberOfPlayer` has a quote character? :nerd_face:

File: pacman-server/src/pages/admin.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
+<!doctype html>
+<html>
+
+<head>
+    <title>Socket.IO chat</title>
+    <style>
```
Comment (nthungvn) [inline pacman-server/src/pages/admin.html]: Can we move style to separate file?

File: pacman-server/src/pacman.js (hunk 1 above)
Comment (phuchieuct93abc) [reply pacman-server/src/pacman.js]: corrected

File: pacman-server/src/pages/admin.html (hunk 2 above)
Comment (phuchieuct93abc) [reply pacman-server/src/pages/admin.html]: corrected

**Pull request merged**